<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* Groupbox */

QGroupBox {
    border: 1px solid #DDD;
    border-radius: 9px;
    margin-top: 9px;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>6</number>
    </property>
    <property name="topMargin">
     <number>6</number>
    </property>
    <property name="rightMargin">
     <number>6</number>
    </property>
    <property name="bottomMargin">
     <number>6</number>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>230</width>
        <height>0</height>
       </size>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane {
    background: transparent;
    border:0;
}
</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Interface</string>
       </attribute>
       <widget class="QGroupBox" name="PersistenceGroupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>201</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>Persistence</string>
        </property>
        <widget class="QPushButton" name="Load">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Load</string>
         </property>
        </widget>
        <widget class="QPushButton" name="Save">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>20</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
        <widget class="QPushButton" name="loadTriangleButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Triangle</string>
         </property>
        </widget>
        <widget class="QPushButton" name="loadQuadButton">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>50</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Quad</string>
         </property>
        </widget>
        <widget class="QPushButton" name="loadPentagonButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Pentagon</string>
         </property>
        </widget>
        <widget class="QPushButton" name="loadHexagonButton">
         <property name="geometry">
          <rect>
           <x>100</x>
           <y>70</y>
           <width>91</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string>Hexagon</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="RenderingGroupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>201</width>
          <height>151</height>
         </rect>
        </property>
        <property name="title">
         <string>Rendering</string>
        </property>
        <widget class="QSpinBox" name="SubdivSteps">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLabel" name="StepsLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>101</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Catmull-Clark steps</string>
         </property>
        </widget>
        <widget class="QComboBox" name="Renderer">
         <property name="geometry">
          <rect>
           <x>80</x>
           <y>20</y>
           <width>111</width>
           <height>21</height>
          </rect>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="RendererLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Renderer</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="TessellationLevel">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>70</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="minimum">
          <number>1</number>
         </property>
         <property name="maximum">
          <number>64</number>
         </property>
         <property name="value">
          <number>50</number>
         </property>
        </widget>
        <widget class="QLabel" name="TessellationLevelLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Tessellation level</string>
         </property>
        </widget>
        <widget class="QLabel" name="ColorBandsLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>61</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Color bands</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="ColorBands">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="minimum">
          <number>0</number>
         </property>
         <property name="maximum">
          <number>50</number>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
        <widget class="QCheckBox" name="ShowWireframe">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>120</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="ShowWireframeLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>120</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Show wireframe</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>420</y>
          <width>201</width>
          <height>141</height>
         </rect>
        </property>
        <property name="title">
         <string>Difference Limit Surface</string>
        </property>
        <widget class="QCheckBox" name="DisplayDifference">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
        <widget class="QLabel" name="DisplayDifferenceLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>91</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Display difference</string>
         </property>
        </widget>
        <widget class="QLabel" name="DiffStepsLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>181</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Limit CC steps</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="DiffSubdivSteps">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
        </widget>
        <widget class="QLabel" name="DiffScaleLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Difference scale</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="DiffScale">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>70</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximum">
          <number>1000</number>
         </property>
         <property name="singleStep">
          <number>1</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
        </widget>
        <widget class="QLabel" name="ColormapPicture">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>130</y>
           <width>181</width>
           <height>8</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;colormap picture&gt;</string>
         </property>
         <property name="scaledContents">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QLabel" name="ColormapMaxLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>110</y>
           <width>191</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;colormap max&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="MaxDiff">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>90</y>
           <width>81</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt; image max&gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
        <widget class="QLabel" name="MaxDiffLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>90</y>
           <width>111</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Max visible difference</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="EditingGroupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>280</y>
          <width>201</width>
          <height>131</height>
         </rect>
        </property>
        <property name="title">
         <string>Editing</string>
        </property>
        <widget class="QLabel" name="EditStepsLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>50</y>
           <width>51</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Edit steps</string>
         </property>
        </widget>
        <widget class="QSpinBox" name="EditSteps">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>50</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="maximum">
          <number>6</number>
         </property>
        </widget>
        <widget class="QCheckBox" name="EnableEditing">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>20</y>
           <width>21</width>
           <height>23</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="EditColorButton">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>100</y>
           <width>61</width>
           <height>21</height>
          </rect>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
        <widget class="QLabel" name="EnableEditingLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Enable editing</string>
         </property>
        </widget>
        <widget class="QLabel" name="BrushRadiusLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>70</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Brush radius</string>
         </property>
        </widget>
        <widget class="QDoubleSpinBox" name="BrushRadiusSpinBox">
         <property name="geometry">
          <rect>
           <x>130</x>
           <y>70</y>
           <width>61</width>
           <height>22</height>
          </rect>
         </property>
         <property name="singleStep">
          <double>0.010000000000000</double>
         </property>
         <property name="value">
          <double>0.200000000000000</double>
         </property>
        </widget>
        <widget class="QLabel" name="BrushColorLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>100</y>
           <width>71</width>
           <height>20</height>
          </rect>
         </property>
         <property name="text">
          <string>Brush color</string>
         </property>
        </widget>
       </widget>
       <widget class="QPushButton" name="screenShot">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>670</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>ScreenShot</string>
        </property>
       </widget>
       <widget class="QPushButton" name="background">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>610</y>
          <width>112</width>
          <height>32</height>
         </rect>
        </property>
        <property name="text">
         <string>BackGround</string>
        </property>
       </widget>
       <widget class="QDoubleSpinBox" name="scaleSpinBox">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>610</y>
          <width>62</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Information</string>
       </attribute>
       <widget class="QGroupBox" name="InfoGroupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>130</y>
          <width>201</width>
          <height>241</height>
         </rect>
        </property>
        <property name="title">
         <string>Info</string>
        </property>
        <widget class="QLabel" name="InfoLabel">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>181</width>
           <height>211</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt; info &gt;</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="ShortcutsGroupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>201</width>
          <height>111</height>
         </rect>
        </property>
        <property name="title">
         <string>Shortcuts</string>
        </property>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>91</width>
           <height>91</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
Pan&lt;br&gt;
Zoom&lt;br&gt;
Edit level&lt;br&gt;
Edit coords&lt;br&gt;
Edit color&lt;br&gt;
Delete edit&lt;br&gt;
&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
        <widget class="QLabel" name="label_2">
         <property name="geometry">
          <rect>
           <x>70</x>
           <y>20</y>
           <width>131</width>
           <height>91</height>
          </rect>
         </property>
         <property name="text">
          <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;
( left drag )&lt;br&gt;
( scroll )&lt;br&gt;
( ctrl + scroll )&lt;br&gt;
( ctrl + left drag )&lt;br&gt;
( ctrl + right drag )&lt;br&gt;
( shift + edit )&lt;br&gt;
&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="MainView" name="MainDisplay">
      <property name="mouseTracking">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainView</class>
   <extends>QOpenGLWidget</extends>
   <header>mainview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
